packages/acs-ui-common/package.json

```diff
@@ -33,7 +33,8 @@
   },
   "peerDependencies": {
     "react": ">=16.8.0 <18.0.0",
-    "@types/react": ">=16.8.0 <18.0.0"
+    "@types/react": ">=16.8.0 <18.0.0",
+    "@azure/communication-calling": "1.8.0-beta.1 || >=1.4.4"
```

Comment:
my understanding is acs-ui-common should only be for common utils/types used across both calling and chat. We have a dependency chain within calling where stateful calling packlet is the base dependency that can export these. Otherwise Chat now has an implicit dependency on communication-calling

Reply:
These should be exported as `@internal` symbols from `calling-component-bindings` instead of `acs-ui-common`.

Reply:
(Or `calling-stateful-client`, depends on where it is needed.

Reply:
We are now export everything by 
```
exporting * from 'call-stateful-client'
```

Then we might need to change this pattern a bit to only export types/functions we need, what do you think?

Reply:
Looking at composite package, now following the pattern of index-public.ts to resolve this problem